src/lib.rs

```diff
@@ -242,12 +242,10 @@ impl TOTP {
     /// ```
     /// * `digits`: MUST be between 6 & 8
     /// * `secret`: Must have bitsize of at least 128
-    /// * `account_name`: Must not contain `:`
-    /// * `issuer`: Must not contain `:`
```

Comment:
I'm sorry to seem insistent on this, but this is standard per optauth uri format. Your example URL respects this

Reply:
I have reverted this. Along with that I've fixed the `new_unchecked` doctests which were failing.